.github/workflows/desmos_bindings.yml

```diff
@@ -69,11 +69,9 @@ jobs:
           components: rustfmt, clippy
 
       - name: Run cargo fmt
-        working-directory: ./packages/bindings
         run: cargo fmt --all -- --check
 
       - name: Run cargo clippy
-        working-directory: ./packages/bindings
```

Comment:
Set working-directory to work-place can check `cargo clippy` errors in this repo